src/wwwroot/partials/settings/my-plan.html

```diff
@@ -41,7 +41,8 @@
       </div>
       <div class="flex two flex--wrap">
         <label class="width--full special uppercase">{{ 'my_plan_renewal_date' | translate }}</label>
```

Comment:
I think that this label will also change. There are three different scenarios:

1. When `accountEndDate` has value and _the account is free_ it should say "Free trial end date" (or something similar, please confirm)
2. When `accountEndDate` has value and _the account is NOT free_ it should say "Account end date" (or something similar, please confirm)
3. When `accountEndDate` do not have value, it should say "Renewal date" (as now)

Of course, the value should be related to the label.